mani_skill/examples/benchmarking/gpu_sim.py

```diff
@@ -20,6 +20,8 @@
 @dataclass
 class Args:
     env_id: Annotated[str, tyro.conf.arg(aliases=["-e"])] = "PickCube-v1"
+    robot_uids: Annotated[str, tyro.conf.arg(aliases=["-r"])] = "widowxai"
+    """Some environments support multiple robots"""
```

Comment:
maybe make this more informative like

"Pass in a robot uid to change the robot loaded into the environment instead of using the default one"